doc/src/site/sphinx/ml/sw_deep_learning.rst

```diff
@@ -0,0 +1,114 @@
+        or the layout of hidden layers.
+
+        .. code:: scala
+
+            import ai.h2o.sparkling.ml.algos.H2ODeepLearning
+            val estimator = new H2ODeepLearning().setLabelCol("CAPSULE").setHidden(Array(3, 2))
+            val model = estimator.fit(trainingDF)
+
+        By default, the ``H2ODeepLearning`` algorithm distinguishes between a classification and regression problem based on the type of
+        the label column of the training dataset. If the label column is a string column, a classification model will be trained.
+        If the label column is a numeric column, a regression model will be trained. If you don't want be worried about
+        column data types, you can explicitly identify the problem by using ``ai.h2o.sparkling.ml.algos.classification.H2ODeepLearningClassifier``
+        or ``ai.h2o.sparkling.ml.algos.regression.H2ODeepLearningRegressor`` instead.
+
+        You can also get raw model details by calling the *getModelDetails()* method available on the model as:
+
+        .. code:: scala
+
+            model.getModelDetails()
+
+        Run Predictions
+
+        .. code:: scala
+
+            model.transform(testingDF).show(false)
+
+
+    .. tab-container:: Python
+        :title: Python
+
+        First, let's start PySparkling Shell as
+
+        .. code:: shell
+
+            ./bin/pysparkling
+
+        Start H2O cluster inside the Spark environment
+
+        .. code:: python
+
+            from pysparkling import *
+            hc = H2OContext.getOrCreate()
+
+        Parse the data using H2O and convert them to Spark Frame
+
+        .. code:: python
+
+            import h2o
+            frame = h2o.import_file("https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/prostate/prostate.csv")
+            sparkDF = hc.asSparkFrame(frame)
+            sparkDF = sparkDF.withColumn("CAPSULE", sparkDF.CAPSULE.cast("string"))
+            [trainingDF, testingDF] = sparkDF.randomSplit([0.8, 0.2])
+
+        Train the model. You can configure all the available Deep Learning arguments using provided setters or constructor parameters,
+        such as the label column or the layout of hidden layers.
+
+        .. code:: python
+
+            from pysparkling.ml import H2ODeepLearning
+            estimator = H2ODeepLearning(labelCol = "CAPSULE", hidden=[3, 2])
+            model = estimator.fit(trainingDF)
+
+        By default, the ``H2ODeepLearning`` algorithm distinguishes between a classification and regression problem based on the type of
+        the label column of the training dataset. If the label column is a string column, a classification model will be trained.
+        If the label column is a numeric column, a regression model will be trained. If you don't want to be worried about
+        column data types, you can explicitly identify the problem by using ``H2ODeepLearningClassifier`` or ``H2ODeepLearningRegressor`` instead.
```

Comment:
Change
 
you can explicitly identify the problem by

to 

you can explicitly specify the model by